src/components/vertical-divider/vertical-divider.style.ts

```diff
@@ -1,18 +1,19 @@
 import styled, { css } from "styled-components";
 import { space } from "styled-system";
+import { VerticalDividerPropTypes } from "./vertical-divider.component";
 import { baseTheme } from "../../style/themes";
 
-const StyledVerticalWrapper = styled.div`
+const StyledVerticalWrapper = styled.div<VerticalDividerPropTypes>`
   ${space}
   ${({ height }) =>
     height &&
     css`
-      height: ${typeof height === "string" ? `${height};` : `${height}px;`};
+      height: ${typeof height === "string" ? `${height}` : `${height}px`};
     `}
   ${({ displayInline }) => (displayInline ? "display: inline;" : "")}
 `;
 
-const StyledDivider = styled.div`
+const StyledDivider = styled.div<Partial<VerticalDividerPropTypes>>`
```

Comment:
This only needs the `tint` prop from `VerticalDividerPropTypes` - let's use `Pick` to grab only that one from `VerticalDividerPropTypes`